content/en/docs/tutorials/hello-minikube.md

```diff
@@ -96,6 +96,11 @@ Pod runs a Container based on the provided Docker image.
     ```shell
     kubectl create deployment hello-node --image=registry.k8s.io/echoserver:1.4
     ```
+    
+For M1 laptops you could run:
```

Comment:
Thanks. We should specify a broader context; this applies to any computer that uses AArch64 CPUs and not just a specific kind of laptop.

We should _also_ fix the echoserver image; see https://github.com/kubernetes/website/pull/36731#issuecomment-1250246299 and other comments on that PR.